### PR TITLE
Added integration for SourceEditor app with new screen manager SDK

### PR DESCRIPTION
**Pull request by ancirja-m**

### CODE REVIEW

File: SourceEditor/dependencies.gradle [hunk 1]
```diff
@@ -16,10 +16,9 @@ ext {
     duo_url = "https://pkgs.dev.azure.com/MicrosoftDeviceSDK/DuoSDK-Public/_packaging/Duo-SDK-Feed/maven/v1"
 
     config = [
+            duoSdkVersion            : "1.0.0-beta1",
```
Comment (CesarValiente) [inline SourceEditor/dependencies.gradle:19]: I believe there are already other PRs that handle this, so you will probably have conflicts with this ;-)

**Review by CesarValiente — APPROVED** [review]
LGTM! Thanks Andrei!

File: SourceEditor/dependencies.gradle (hunk 1 above)
Comment (ancirja-m) [reply SourceEditor/dependencies.gradle:19]: Actually no, because each app has his own dependencies file.

**Review by bimiron — APPROVED** [review]

File: SourceEditor/dependencies.gradle (hunk 1 above)
Comment (CesarValiente) [reply SourceEditor/dependencies.gradle:16]: wops!! true that! I thought it was like in the kotlin samples. So many PRs today! 😅🤣

**Pull request merged**